CodeGeneration/Sources/Utils/Utils.swift

```diff
@@ -10,39 +10,36 @@
 //
 //===----------------------------------------------------------------------===//
 
-/// Trims leading and trailing whitespace from each line.
-public func dedented<Lines: Sequence>(lines: Lines) -> [String] where Lines.Element: StringProtocol {
-  lines.map { $0.trimmingCharacters(in: .whitespacesAndNewlines) }
-}
-
-/// Trims leading and trailing whitespace from each line.
-public func dedented(string: String) -> String {
-  dedented(lines: string.split(separator: "\n"))
-    .joined(separator: "\n")
+extension Sequence where Element: StringProtocol {
+  /// Trims leading and trailing whitespace from each line.
+  var dedented: [String] {
```

Comment:
`dedented` doesn't really imply trim leading + trailing to me. We use `trimmed` for nodes but... 🤷

Reply:
Oh, the `dedented` is still a leftover from the Python days. We can just remove it completely.